docs/notebooks/point_clouds.ipynb

```diff
@@ -13,30 +13,28 @@
   },
```

Comment:
<p><code>Norm.</code> -&gt; <code>norm</code> </p><p>Would also reformulate <code>but play with these settings at your own risk!</code>, because it sounds too harsh I think.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
yes! thanks.